embedding-calculator/src/docs/find_faces_base64_post.yml

```diff
@@ -0,0 +1,91 @@
+tags:
+  - Core
+summary: 'Find faces in the given image in BASE64 format and return their bounding boxes.'
+description: 'Returns bounding boxes of detected faces on the image.'
+operationId: findFacesBase64Post
+consumes:
+  - multipart/form-data
```

Comment:
It definitely should consume `application/json`
Please also check if this swagger is working. To check, just open:
http://localhost:3300/apidocs
in browser, find the needed rest endpoint and try to invoke it using swagger

Reply:
Checked. It is working